test-prow-e2e.sh

```diff
@@ -13,6 +13,32 @@ function copyArtifacts {
   fi
 }
 
+function waitForAuthOperatorProgressing {
+  output=''
+  count=0
+  sleepDuration=15
+  maxRetries=40  #10 minutes
+  action="started"
+  if [ "$1" == "False" ]; then
+    action="finished"
+  fi
+
+  until [ "$output" == "$1" ] || [ $count -gt $maxRetries ]; do
+    output=$(oc get co/authentication -o 'jsonpath={.status.conditions[?(@.type=="Progressing")].status}')
+    ((count=count+1))
+    sleep $sleepDuration
+  done
+
+  secs=count*sleepDuration;
```

Comment:
Do you mind fixing the shellcheck warnings here?

```
shellcheck ~/Downloads/test-prow-e2e.sh

In /Users/sam/Downloads/test-prow-e2e.sh line 32:
  secs=count*sleepDuration;
       ^-----------------^ SC2100: Use $((..)) for arithmetics, e.g. i=$((i * 2))
       ^-----------------^ SC2125: Brace expansions and globs are literal in assignments. Quote it or use an array.

For more information:
  https://www.shellcheck.net/wiki/SC2100 -- Use $((..)) for arithmetics, e.g....
  https://www.shellcheck.net/wiki/SC2125 -- Brace expansions and globs are li...
```

Reply:
fixed